perun-registrar-lib/src/main/java/cz/metacentrum/perun/registrar/impl/AppAutoRejectionScheduler.java

```diff
@@ -46,6 +50,27 @@
 	private static final String A_VO_APP_EXP_RULES = "urn:perun:vo:attribute-def:def:applicationExpirationRules";
 	private static final String A_GROUP_APP_EXP_RULES = "urn:perun:group:attribute-def:def:applicationExpirationRules";
 
+	private static final String A_VO_APP_REJECT_MESSAGES = "urn:perun:vo:attribute-def:def:applicationAutoRejectMessages";
+	private static final String A_GROUP_APP_REJECT_MESSAGES = "urn:perun:group:attribute-def:def:applicationAutoRejectMessages";
+
+	private static final String A_USER_PREFERRED_LANGUAGE = "urn:perun:user:attribute-def:def:preferredLanguage";
+
+	private static final String DEFAULT_VO_MSG_ADMIN = "Your application to VO %vo_name% was " +
+			"automatically rejected, because admin didn't approve your application in a timely manner.";
+	private static final String DEFAULT_GROUP_MSG_ADMIN = "Your application to group %group_name% " +
+			"was automatically rejected, because admin didn't approve your application in a timely manner.";
```

Comment:
You could probably use the placeholders here as well.

Reply:
You are right, I forget about that.